source/aws-bootstrap-kit/lib/secure-root-user.ts

```diff
@@ -46,14 +51,89 @@ export class SecureRootUser extends core.Construct {
       }
     );
 
+    // Create role used for auto remediation
+    const autoRemediationRole = new iam.Role(this, 'AutoRemediationRole', {
+      assumedBy: new iam.CompositePrincipal(
+          new iam.ServicePrincipal("events.amazonaws.com"),
+          new iam.ServicePrincipal("ssm.amazonaws.com"),
+      )
+    });
+
+    autoRemediationRole.addManagedPolicy(iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AmazonSSMAutomationRole'));
+
     enforceMFARule.node.addDependency(configRecorder);
     enforceNoAccessKeyRule.node.addDependency(configRecorder);
 
-    enforceMFARule.onComplianceChange('ComplianceChange', {
-      target: new targets.SnsTopic(snsTopic)
-    });
-    enforceNoAccessKeyRule.onComplianceChange('ComplianceChange', {
-      target: new targets.SnsTopic(snsTopic)
+    secureRootUserConfigTopic.grantPublish(autoRemediationRole);
+
+    // Create remediations by notifying owner
+
+    new config.CfnRemediationConfiguration(this, "enforceMfaNotification", {
```

Comment:
Both configs have common properties. Shall we extract this to a function to reduce duplication?

Reply:
Will do